src/main/resources/webroot/main.js

```diff
@@ -185,17 +192,69 @@ angular
         vm.vertxProject.vertxDependencies = [];
         vm.vertxProject.packageName = "";
         vm.vertxProject.jdkVersion = defaults.jdkVersion;
+
+        vm.disableDependencies(defaults.vertxVersion);
+        refreshAvailableDependencies(defaults.vertxVersion, true);
+      }
+
+      function refreshAvailableDependencies(vertxVersion, initializing) {
+        vm.availableVertxDependencies = availableDependencies(vertxVersion, initializing)
+      }
+
+      function availableDependencies(version, initializing) {
```

Comment:
How about merging this one with `refreshAvailableDependencies`?

Reply:
👷 Done